icechunk/src/format/mod.rs

```diff
@@ -150,7 +150,7 @@ pub enum ByteRange {
     Bounded(Range<ChunkOffset>),
     /// All bytes from the given offset (included) to the end of the object
     From(ChunkOffset),
-    /// Last n bytes in the object
+    /// All bytes up to the last n bytes in the object
     Last(ChunkLength),
```

Comment:
```suggestion
    Until(ChunkLength),
```

i think we should rename, last is just confusing now.